src/test/java/com/ericsson/eiffel/remrem/semantics/ValidationTest.java

```diff
@@ -63,5 +63,21 @@ public void TestInvalidActivityFinishedOutput() throws Exception {
         
     }
 
-    
+    @Test
+    public void TestEventwithCauseAndContextLinkTypes() throws Exception {
+        try {
+            File file = new File("output/TriggeredwithCauseAndContext.json");
+            if (file.exists()) {
+                JsonObject object = parser.parse(new FileReader(file)).getAsJsonObject();
+                String msgType = object.get("meta").getAsJsonObject().get("type").getAsString();
+                EiffelValidator validator = EiffelOutputValidatorFactory
+                        .getEiffelValidator(EiffelEventType.fromString(msgType));
+                validator.validate(object);
+                validator.customValidation(object);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+            Assert.assertFalse(true);
```

Comment:
What exception is this?  If the exception came, does it mean test pass?

Reply:
If the event validation fails, it will come to the exception block. If the exception came, the testcase will fail.

Reply:
Ok, But why Assert.assertFalse(true);?

Reply:
with this assert check, if exception came the testcase will fail.
without this assert check, if exception came the testcase will pass.

Reply:
ok